R/multiSample.R

```diff
@@ -303,16 +303,28 @@ mergeSequenceTables <- function( table1=NULL, table2=NULL, ..., tables=NULL, rep
 		tables <- list(table1, table2)
 		tables <- c(tables, list(...))
 	}
+ 
   # Validate tables
-  if(!(all(sapply(tables, is.sequence.table)))) {
-    stop("At least two valid sequence tables, and no invalid objects, are expected.")
+  if(length(tables)<2){
+    stop("At least two sequence tables are expected")
+  }
+  tablesValid <- sapply(tables, is.sequence.table)
+  if(!(all(tablesValid))) {
+    errorMessage <- paste0(names(tables[which(!tablesValid)]), collapse=", ")
+    if(length(errorMessage)){
```

Comment:
Can you clarify the purpose of this condition?
It will trigger as long as `length(errorMessage) > 0`, but won't that always be the case?
Even if names are NULL, you still get an empty string, which is a character vector of length 1.
Should the check here be using `nchar` instead?

Reply:
It's not very useful :) but the message is in good grammar when there isn't a string.

I've checked that `length(paste0(NULL))` returns 0 but you're right that it should be nchar, that's what I meant:
```
> length("pancake")
[1] 1
> nchar("pancake")
[1] 7
```
Thank you!